### DISCUSSION TITLE
Galaxy Brain Badge

### DISCUSSION THREAD

**Discussion by ws-jm** [community/community#39302] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hope get this badge.
![image](https://user-images.githubusercontent.com/117026294/202382707-42dba021-a994-471c-89c5-119e4d340e3c.png)

Let's start!

**Comment by DSDmark** [comment, marked as answer]
Hi @ws-jm,

Check this out, there is a whole discussion about that achievement :- (https://github.com/community/community/discussions/18293).

> And I suggest you if you want any question then, don't just create discussion. search first, there are lots of question which is already answered by someone. It's save you time to get your desire answer.

**Reply by ws-jm** [reply]
Thanks DSDmark.

**Comment by lawcompanion1** [comment]
Thanks.
I will try.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Pavel-Shcherbo** [comment]
How do I get a Galaxy Brain badge if I have answered 20 discussions?

**Reply by Jonak-Adipta-Kalita** [reply]
it needs to be "marked as answer"

**Reply by saifeddineelhanoune** [reply]
but the question is how can i mark a comment as answer? i can't find any button indexing this thing

**Comment by yusrilprayoga-code** [comment]
thanks
